### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by hu-jeff**

### CODE REVIEW

**Review by ec105 — APPROVED** [review]

**Review by SamandarVijay — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
-# User-specific stuff
```
Comment (is1006) [inline .gitignore:4]: why is this red

**Review by ec105 — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (is1006) [reply .gitignore:4]: nevermind

**Review by is1006 — APPROVED** [review]

File: src/main/java/tutorial/User.java [hunk 2]
```diff
@@ -15,11 +22,15 @@ public class User {
      * @param id the User's id as assigned when the User is created
      */
 
-    public User(String displayName, int id) {
+    public User(String displayName, int id, String username, String password) {
         this.displayName = displayName;
-        this.id = id; //need to determine how we will be creating id and ensuring it does not already exist //
```
Comment (ec105) [inline src/main/java/tutorial/User.java:20]: we have chaned to make username the idential compoent, where we will be checking no existed username will be created in CreateAccountPage

**Review by ec105 — APPROVED** [review]

**Review by SamandarVijay — APPROVED** [review]
Looks Good friend

**Review by is1006 — APPROVED** [review]

**Pull request merged**